### DISCUSSION TITLE
What effect is this in the Windows Store app and is it possible in WinUI 3?

### DISCUSSION THREAD

**Discussion by yki5000** [microsoft/microsoft-ui-xaml#9496] [Q&A]
Hi,
is this custom made in UWP and is it possible in WinUI 3?

https://github.com/microsoft/microsoft-ui-xaml/assets/70078713/7313d1ef-73e7-4184-aad4-498c4df5fee5

Is this possibly something similar to the "Animation interop" from the WinUI 3 Gallery and then something is faded in, what is the procedure for fading in?

**Comment by ghost1372** [comment]
I am also interested in these things😁
Animations are most likely created with Composition
and Image Slider is a Custom control and is not (FlipView + PipsPager)

**Comment by JustinXinLiu** [comment]
Thanks for your interest in this. :)

Our team (Microsoft Store) spent tons of time refining this type of experiences and we already made most of it open source. Please take a look at [this link](https://github.com/CommunityToolkit/Labs-Windows/issues/428).

**Reply by Fritierkelle** [reply]
The control/ FlipView with the PipsPager in the video would be great, if this was already implemented or something like a horizontal carousel/ GridView with PipsPager etc.

**Comment by LudoTexx** [comment]
Did you have checked the windows Community Toolkit Gallery app on the MS store : 

https://apps.microsoft.com/detail/9nblggh4tlcq?hl=en-us&gl=US

Look for Connected animations, Implicit animations and effect animations.

**Reply by Fritierkelle** [reply]
Would be nice if this were also available for c++.
